src/index.js

```diff
@@ -91,7 +91,9 @@ function useWalletBalance({ account, ethereum, pollBalanceInterval }) {
       return {
         async request() {
           return getAccountBalance(ethereum, account)
-            .then(value => (value ? JSBI.BigInt(value).toString() : NO_BALANCE))
+            .then((value) =>
```

Comment:
Oh, do we want to keep the new parenthesis default for prettier?

Reply:
Ah yes, I moved to Prettier 2 while I was upgrading other dependencies, and we want to follow the defaults ([see this discussion](https://github.com/aragon/network-dashboard/pull/2#discussion_r465670884)).

Reply:
Yep! I definitely agree on aligning with defaults. (Even though I do think less parentheses are more elegant :))